packages/office-ui-fabric-react/src/components/Button/CompoundButton/CompoundButton.styles.ts

```diff
@@ -34,12 +34,25 @@ export const getStyles = memoizeFunction((
     },
 
     flexContainer: {
-      flexDirection: 'column',
+      flexDirection: 'row',
       alignItems: 'flex-start',
       minWidth: '100%',
       margin: ''
     },
 
+    textContainer: {
+      textAlign: 'left'
+    },
+
+    icon: {
+      float: 'left',
```

Comment:
can we use flex instead of float? This makes the component much less fragile, and easy to modify.  can then remove lineheight and height and use flex container to stretch to fit.